io/src/FileSystem/private/GoogleCloudStorage_p.cpp

```diff
@@ -50,15 +50,23 @@ bool GoogleCloudStorage::Private::connect(const FileSystemConnection & fileSyste
 		fileSystemConnection.getConnectionProperty(ConnectionProperty::USE_DEFAULT_ADC_JSON_FILE);
 	const std::string adcJsonFile = fileSystemConnection.getConnectionProperty(ConnectionProperty::ADC_JSON_FILE);
 
-	google::cloud::StatusOr<gcs::ClientOptions> opts = gcs::ClientOptions::CreateDefaultClientOptions();
-	if(!opts) {
-		const std::string error =
-			"Couldn't create gcs::ClientOptions for Project ID " + projectId + " status=" + opts.status().message();
-		std::cerr << error << std::endl;
-		throw std::runtime_error(error);
-	}
-
-	auto connConf = opts->set_project_id(projectId);
+	google::cloud::StatusOr<std::shared_ptr<gcs::oauth2::Credentials>> credentials;
+    if (useDefaultAdcJsonFile == "true") {
```

Comment:
If `useDefaultAdcJsonFile == "true"`  then where does `gcs::oauth2::GoogleDefaultCredentials()` point to? can we find out?  Is that set by the environment variable:
`export GOOGLE_APPLICATION_CREDENTIALS="KEY_PATH"`

Reply:
Yes  ` gcs::oauth2::GoogleDefaultCredentials() ` points to  `GOOGLE_APPLICATION_CREDENTIALS` variable. 
 So we can do: 
 `export GOOGLE_APPLICATION_CREDENTIALS="<adc_json_file.json>"`